locals.tf

```diff
@@ -0,0 +1,30 @@
+locals {
+  # Terraform module and the associated app version are tightly coupled for compatibility. 
+  # Specified 'image_version' corresponds to a tested combination of Terraform and app release.
+  # Avoid manually changing the 'image_version' unless you have explicit instructions to do so.
+  image_version_console    = "v7.06.003"
+  image_version_agent      = "v7.06.001"
+  ecr_account              = coalesce(var.ecr_account, "${local.aws_region}" == "us-gov-west-1" ? "822167061992" : "564477214187")
```

Comment:
I think we need to check AWS partition for the Gov partition instead here, since they could deploy in `us-gov-east-1` as well.